src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -25,6 +25,10 @@ the MIT License. See LICENSE in the project root for license information. -->
                 <TextBlock Text="{x:Bind EnumName, Mode=OneWay}"/>
             </DataTemplate>
 
+            <DataTemplate x:DataType="x:String" x:Key="FontFaceComboBoxItemTemplate">
+                <TextBlock Text="{x:Bind}" FontFamily="{x:Bind}"/>

```

Comment:
Did you double check that this is fine receiving the localized name on other languages? I know you're keeping the localized name in the list for sorting and presentation.... but I have no idea what code TextBlock is calling to turn that localized string into its display font. I just want to make sure you double checked this on at least one non-English scenario.

Reply:
I don't know how to test this, but I have a feeling that this is definitely going to fail. Working on a solution rn.